gettsim/pre_processing/piecewise_functions.py

```diff
@@ -28,27 +28,42 @@ def piecewise_polynomial(
         The value of `x` under the piecewise function.
 
     """
+    # If no individual is transferred, we return an empty series
+    if x.empty:
+        return x
+    # # Check if missing data is given
```

Comment:
Delete comments

Reply:
All comments or you mean the extra #?

Reply:
The commented-out code which I assume is a leftover from debugging